README.md

```diff
@@ -33,11 +55,13 @@ timeout 5
 ```
 
 You'll need to update the IP address to match the running instance's IP address.
-Use ```minikube ip`` to get the address
+Use `minikube ip` to get the address
 
 Docs for Ingress-dns are here
 https://github.com/kubernetes/minikube/tree/master/deploy/addons/ingress-dns
 
-### 413 Errors
+#### 413 Errors with Nginx
+
 The default setting for Nginx allows for very small upload sizes. Add this annotation to the ingress for each product to remove teh limit:
-```nginx.ingress.kubernetes.io/proxy-body-size: "0"```
+
+`nginx.ingress.kubernetes.io/proxy-body-size: "0"`
```

Comment:
out of curiosity, why remove the triple-quotes? Doesn't the block-form of code quotes make it easier to copy & paste?

Reply:
I was regarding the difference as single-line vs multi-line, so switched them to singles, since there was only one line. I just went back to triples to see if it rendered any differently than with singles when it's just one line, and either way appeared to render the same way.
I guess, I _do_ mean these to be rendered as their own block, though, even as a single line, so I'll do proper fenced triple quotes. https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting
thanks for pointing it out.

Reply:
clarification...inline triples are the same as inline singles. multi-line triples have a bit more of a block styling.